index.js

```diff
@@ -1,25 +1,33 @@
 // @flow
 
 import React from 'react';
-import { Image } from 'react-native';
+import { Image, TouchableOpacity } from 'react-native';
 import * as flags from './flags';
 
 type Props = {
   size: 16 | 24 | 32 | 48 | 64,
   code: string,
   type?: 'flat' | 'shiny',
   style?: any,
+  onPress: void
 };
 
-const Flag = ({ size = 64, code, type = 'shiny', style }: Props) => {
+const Flag = ({ size = 64, code, type = 'shiny', style, onPress }: Props) => {
   const flag = flags[type][`icons${size}`][code];
   const unknownFlag = flags[type][`icons${size}`]['unknown'];
 
-  return (
+  const flagImage =
     <Image
       source={flag || unknownFlag}
       style={[{ width: size, height: size }, style]}
     />
+  
+  if (!onPress) return flagImage;
+  
+  return (
+    <TouchableOpacity onPress={onPress}>
```

Comment:
I don't suppose it's a good idea to wrap every flag into TouchableOpacity. It's a great redundant bias